styles/controllers/rangeSlider.less

```diff
@@ -11,24 +11,85 @@
  */
 
 .pkp_controllers_rangeSlider {
-	.pkp_controllers_rangeSlider_sliderLabel {
-		margin-bottom: 10px;
-		font-size: 80%;
+
+	.ui-slider {
+		position: relative;
+		background: #bbb;
+		border-radius: 2px;
+		text-align: left;
+		cursor: pointer;
 	}
 
-	.pkp_controllers_rangeSlider_sliderValue {
-		border: 0pt none;
-		font-weight: bold;
-		width: 50px;
+	.ui-slider-horizontal {
+		height: 4px;
+		margin: @half;
 	}
 
-	.pkp_controllers_rangeSlider_slider {
-		margin-left: 13px;
-		width: 240px;
+	.ui-slider-range {
+		position: absolute;
+		z-index: 1;
+		top: 0;
+		height: 4px;
+		background: #fff;
+		background-image: linear-gradient(to right, #ccc, #fff);
+		border: @bg-border;
 	}
 
-	label {
-		display: inline;
-		margin-right: 5px;
+	.ui-slider-handle {
+		display: block;
+		position: absolute;
+		top: -6px;
+		z-index: 2;
+		background: #fff;
+		border: @bg-border;
+		width: @base;
+		height: @base;
+		margin-left: -@half;
+		background-image: url('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgi…gd2lkdGg9IjEwMCUiIGhlaWdodD0iMTAwJSIgZmlsbD0idXJsKCNncmFkKSIgLz48L3N2Zz4g');
```

Comment:
To be honest, we can probably get rid of it. I pulled this style out from a codepen and the `background-image` is a fallback for browsers that don't support linear gradients (IE 9). It's probably not clear to subsequent maintainers that it would need to change if the style changed. So probably not a great idea. And IE 9 users will still see a flat white handle, which won't be terrible.

Note to self: there's another base-64 encoded image below in the :after element.